aspnet/security/introduction-to-aspnet-identity.rst

```diff
@@ -0,0 +1,100 @@
+Introduction to ASP.NET Identity
+================================
+
+By `Pranav Rastogi <http://www.asp.net/identity/overview/getting-started/introduction-to-aspnet-identity#author-37788>`_, `Rick Anderson <http://www.asp.net/identity/overview/getting-started/introduction-to-aspnet-identity#author-20106>`_, `Tom Dykstra <http://www.asp.net/identity/overview/getting-started/introduction-to-aspnet-identity#author-36250>`_ and `Jon Galloway <http://www.asp.net/identity/overview/getting-started/introduction-to-aspnet-identity#author-5506f>`_
+
+The ASP.NET membership system was introduced with ASP.NET 2.0 back in 2005, and since then there have been many changes in the ways web applications typically handle authentication and authorization. ASP.NET Identity is a membership system that is used for building modern applications for the web, phone, or tablet.
+
+Getting started with ASP.NET Identity
+-------------------------------------
+ 
+ASP.NET Identity is used in the Visual Studio project templates for ASP.NET `MVC <http://www.asp.net/mvc>`_, `Web Forms <http://www.asp.net/web-forms>`_, `Web API <http://www.asp.net/web-api>`_ and `SPA <http://www.asp.net/single-page-application>`_. In this walkthrough, we'll illustrate how the ASP.NET 5 project templates use ASP.NET Identity to add functionality to register, log in and log out a user. 
+
+The purpose of this article is to give you a high level overview of ASP.NET Identity; you can follow it step by step or just read the details. For more detailed instructions about creating apps using ASP.NET Identity, see the Next Steps section at the end of this article.
+
+1. Create an ASP.NET MVC application with Individual Accounts. You can use ASP.NET Identity in an ASP.NET MVC application, as well as other ASP.NET frameworks, such as Web Forms, Web API, SignalR, etc. In this article, you will start with an ASP.NET MVC application. In Visual Studio, select **File** -> **New** -> **Project**. Then, select the **ASP.NET Web Appication** from the **New Project** dialog box. Continue by selecting a **Web Appication** with **Individual User Accounts** as the authenication method.
```

Comment:
Would be nice to have a bit more conceptual overview with respect to local accounts vs external login providers. This topic is really about local accounts.